package.json

```diff
@@ -36,6 +36,7 @@
     "columnify": "^1.5.1",
     "lodash": "^4.12.0",
     "mkdirp": "^0.5.1",
+    "node-gyp": "~3.4.0",
```

Comment:
so it is possible that we are going to be testing on a version of node that has an older version of npm, and from that an older version of node-gyp... is this going to cause weird inconsistencies?